"Fix compilation of inline assembly with new gcc"

Patch source: http://pkgs.fedoraproject.org/cgit/rpms/memtest86+.git/

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ AS=as -32
 CC=gcc
 
 CFLAGS= -Wall -march=i486 -m32 -O0 -fomit-frame-pointer -fno-builtin \
-	-ffreestanding -fPIC $(SMP_FL) -fno-stack-protector 
+	-ffreestanding -fPIC $(SMP_FL) -fno-stack-protector -fgnu89-inline
 	
 OBJS= head.o reloc.o main.o test.o init.o lib.o patn.o screen_buffer.o \
       config.o cpuid.o linuxbios.o pci.o memsize.o spd.o error.o dmi.o controller.o \
